install-claude-commands.sh

```
@@ -5,18 +5,13 @@
-# Logging functions
-log_info() { echo -e "${BLUE}[INFO]${NC} $1"; }
-log_success() { echo -e "${GREEN}[SUCCESS]${NC} $1"; }
-log_warning() { echo -e "${YELLOW}[WARNING]${NC} $1"; }
-log_error() { echo -e "${RED}[ERROR]${NC} $1"; }
+# Source shared utilities
+UTILI_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/scripts/install_utils.sh"
+if [ ! -f "$UTILI_PATH" ]; then
+    echo "ERROR: Shared utilities not found at $UTILI_PATH"
+    exit 1
+fi
+source "$UTILI_PATH"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)